ui/src/components/Employee/Create/index.tsx

```diff
@@ -89,7 +94,12 @@ const CreateEmployeeForm: React.FC<FormikProps<CreateEmployeeFormValues> & Creat
 
             <div className={c('logo', 'field-container')}>
               <span className={c('field-title')}>Head Shot</span>
-              <ImageUploadField className={c('image-container')} type={'file'} name="headshot" />
+              <ImageUploadField
+                className={c('image-container')}
+                type={'file'}
+                name="headshot"
+                uploadStatus={hadleUploadStatus}
```

Comment:
Why do we need to maintain a separate upload status? Can't we validate on absense of `form.values.headshot`?

Reply:
Image-upload-field is optional field so that's why we need to maintain separate upload status.